HousingFinanceInterimApi/V1/Gateways/AssetGateway.cs

```diff
@@ -16,7 +18,40 @@ public AssetGateway(DatabaseContext context)
 
         public async Task UpdateAssetDetails(UpdateAssetDetailsQuery query, UpdateAssetDetailsRequest request)
         {
-            await _context.UpdateAssetDetails(query, request).ConfigureAwait(false);
+            await UpdateAssetDetailsTable(query, request).ConfigureAwait(false);
+        }
+
+        private async Task UpdateAssetDetailsTable(UpdateAssetDetailsQuery query, UpdateAssetDetailsRequest request)
```

Comment:
At the moment if both `uhPropertyToUpdate` and maPropertyToUpdate` are **both** `null`, this method will just exit successfully without having done anything.

Do we want any specific behaviour for when both of these are `null`?
Like throw an exception and return 404?

Reply:
There wasn't any such validation in the controller to filter out null requests. This simply replaces the logic in the gateway